app/models/concerns/media_page_item.rb

```diff
@@ -7,7 +7,7 @@ module MediaPageItem
 
   def data_from_page_item
     if self.doc.nil?
-      self.doc = self.get_html
+      self.doc = self.get_html({ allow_redirections: :all })
```

Comment:
can we be caught in an infinite redirect loop here?

Reply:
The `allow_redirections` only will allow redirections http => https and https => http. So, I think that we can be caught in an infinite redirect loop in the case where the server is badly configured. I can try to reproduce this case on test

Reply:
OpenUri will raise an error if there is a redirection loop: https://apidock.com/ruby/v1_9_3_392/OpenURI/open_loop/class